uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/tasks-assignments.settings.config.js

```diff
@@ -25,12 +28,15 @@ define([
         templateUrl: config.path.TPL + 'settings.html?v=5'
       });
 
-    $resourceProvider.defaults.stripTrailingSlashes = false;
+    $analyticsProvider.withAutoBase(true);
```

Comment:
This should go to a separate function `configureAnalytics`. Even better if the configuration is stored in some kind of service. Even better would have the configuration in reqangular so it is accessible in T&A and other extensions.

Reply:
1. I can create a separate function, sure
2. I don't think this would work, given this is the `config` phase of the module (notice that we are injecting `$analyticsProvider`, not `$analytics`), and services are available only after it
3. Given that only T&A and L&A out of all the angular apps are actually using this service, i think it would be an overkill for the very small benefit that we would get out of it. And again i'm not even sure how it would work given that this is happening in the `config` phase

Reply:
Please create a separate func. Same for other PR.